apps/pano/app/features/post/MoreOptionsDropdown.tsx

```diff
@@ -69,6 +69,7 @@ export const MoreOptionsDropdown: FC<Props> = ({ post }) => {
     ownerItems.push(<DropdownMenuSeparator key="separator" />);
   }
 
+  // FIXME: below appears to be redundant, is it?
```

Comment:
yeah, this looks redundant but we aim to add more items in the future for this dropdown which won't directly be related with the owner. so keeping it as it is ok with me tbh.